__tests__/app.test.js

```diff
@@ -11,4 +15,62 @@ describe('backend routes', () => {
   afterAll(() => {
     pool.end();
   });
+
+  it('should redirect to the github oauth page upon login', async () => {
+    // a get request that redirects user to github sign in
+    const req = await request(app).get('/api/v1/github/login');
+    // but that github sign in is mocked?
```

Comment:
at this point we are just testing that this endpoint redirects to the correct github url; it's not mocked, but it also doesn't do anything once it's at the github url